ue5osc/__init__.py

```diff
@@ -96,6 +96,18 @@ def get_image(self, filename: str) -> bytes:
         image = Image.open(filename)
         return image
 
+    def console(self, message: str) -> None:
+        """Sends messages to be executed in Unreal Engine's console"""
+        self.client.send_message("/console", message)
+
+    def switch_camera(self) -> None:
+        """Switches between camera views. One where the robot is visible or not"""
+        self.client.send_message("/switch/view", 0.0)

```

Comment:
Is this a toggle command? Or do we send the mode we want it in?

Reply:
It is a toggle command.

Reply:
OK. Can you make that more specific? Maybe like this:

```suggestion
    def toggle_camera_view(self) -> None:
        """Toggles the camera between 1st and 3rd person views."""
        dummy = 0.0
        self.client.send_message("/switch/view", dummy)
```

Reply:
You'll also need to merge my changes into yours before I can accept.